src/services/login.js

```diff
@@ -51,12 +51,20 @@ export const sendResetEmail = async (email) => {
 };
 
 export const checkIfLoggedIn = async () => {
-  const axiosInstance = axiosSub.create();
+  const GERILIFE_SERVER_PROTOCOL =
+    process.env.GERILIFE_SERVER_PROTOCOL || "http";
+  const GERILIFE_SERVER_ADDRESS = process.env.GERILIFE_SERVER_ADDRESS;
+  const GERILIFE_SERVER_PORT = process.env.GERILIFE_SERVER_PORT;
+
+  const API_BASE_URL = `${GERILIFE_SERVER_PROTOCOL}://${GERILIFE_SERVER_ADDRESS}:${GERILIFE_SERVER_PORT}`;
+  const axiosInstance = axiosSub.create({
+    baseURL: API_BASE_URL
+  });
 
   let result;
   try {
     const { data } = await axiosInstance.post(
-      "/api/methods/checkIfLoggedIn?_t=" + new Date().getTime(),
```

Comment:
We need these `/api/` prefixes in order to have a consistent interface to the backend server in our deployment and development environments. Please test/review PR #35